content/blog/2020-12-10-masks-public.Rmd

```diff
@@ -0,0 +1,363 @@
+---
+title: "Are Masks Widely Used in Public?"
+author: "Alex Reinhart"
+date: 2020-12-07
+tags: ["R", "symptom surveys", "COVIDcast"]
```

Comment:
consider using the alternative list format for tags, see https://github.com/cmu-delphi/www-main/blob/5131bbc26916230961e6add2384a4d0184cc0eaa/content/blog/2020-08-26-fb-survey.Rmd#L5-L9